apps/editorial_features/routes.coffee

```diff
@@ -10,16 +10,17 @@ Q = require 'bluebird-q'
 @eoy = (req, res, next) ->
   @curation = new Curation(id: sd.EOY_2016)
   @article = new Article(id: sd.EOY_2016_ARTICLE)
-
   Q.all([
-  	@curation.fetch()
-    @article.fetch()
+    @curation.fetch(cache: true)
+    @article.fetch(
+      cache:  true
```

Comment:
Curious what impact these cache flags have. Curations probably rarely change, so not worried.